src/SWP/Bundle/MultiTenancyBundle/Initializer/PHPCRBasePathsInitializer.php

```diff
@@ -131,6 +132,7 @@ private function createBasePaths(SessionInterface $session, array $basePaths, ar
                 $route = new $this->documentClass();
                 $route->setParentDocument($this->dm->find(null, $path));
                 $route->setName($home);
+                $route->setType(RouteInterface::TYPE_CONTENT);
```

Comment:
this will work in our case because we use our model in config: `document_class: SWP\Bundle\ContentBundle\Doctrine\ODM\PHPCR\Route` and the default one is `SWP\Bundle\MultiTenancyBundle\Document\Page` provided in the `Configuration` class, thus it will fail when using that default class because `Page` doesnt force to implement `getType`, `setType` methods as far as I see.